src/components/fee-summary/SbcFeeSummary.vue

```diff
@@ -130,20 +130,20 @@
 
 <script lang="ts">
 import { Vue, Component, Prop, Watch, Emit } from 'vue-facing-decorator'
-import { FeeServices } from '@bcrs-shared-components/services/'
+import { FeeServices } from '@bcrs-shared-components/services'
 import { Fee, FilingData } from '@bcrs-shared-components/interfaces/'
 
 @Component({})
 export default class SbcFeeSummary extends Vue {
   /* This prop is an array of filing data. See model for details. */
   @Prop({ default: () => [] })
-  filingData!: Array<FilingData>
+    filingData!: Array<FilingData>
```

Comment:
You don't need to indent this (because the decorator is separate from the property declaration).

The 2 ways we normaly style props is
```
  @Prop({ default: () => [] })
  filingData!: Array<FilingData>
```
or
```
  @Prop({ default: () => [] }) filingData!: Array<FilingData>
```

Reply:
It was getting flagged by eslindt yesterday. I'll do the second option

Reply:
ES Lindt sounds delicious 😆